content/news/playtest-20150801.md

```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
+
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
+
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.
+As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
+
+Music playback has received quite a lot of changes. Map authors now have much greater control over music, including the ability to specify default (starting) music and custom victory and defeat tracks.
```

Comment:
default music, custom victory and defeat tracks. This is hacky, and to get the proper sentence you'd need to reword this.